.travis.yml

```diff
@@ -44,4 +44,4 @@ jobs:
     - env: EMBER_TRY_SCENARIO=ember-release
     - env: EMBER_TRY_SCENARIO=ember-beta
     - env: EMBER_TRY_SCENARIO=ember-canary
-    - env: EMBER_TRY_SCENARIO=ember-lts-3.4
```

Comment:
Can you also add a CI run for Ember 3.16? I know it happens to be the current version, but #soon it will not be 😸

Reply:
Thanks for adding the `config/ember-try.js` entry for 3.16, I think you still need to add it here (or it will never run).